server/app/models/place.js

```diff
@@ -16,7 +16,7 @@ var PlaceSchema = new Schema({
     required: true
   },
   photos: [String],
-  likes: [Schema.Types.ObjectId],
+  likes: [String],
```

Comment:
I think it's just more logical solution) If there is an array of id's then it should be of `ObjectId` type